### PR TITLE
PAPP-34667: Fix code quality issues 

### PR DESCRIPTION
**Pull request by tapishj-splunk**
#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The manual_readme_content.md has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe):

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers),
  please document them in the `manual_readme_content.md`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `manual_readme_content.md` all methods (eg, OAuth) used to authenticate
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `manual_readme_content.md`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

## Release Notes (REQUIRED)
- document_commonly_used_service_ports and licenses errors will be fixed with this mr: https://cd.splunkdev.com/phantom/app_release/-/merge_requests/2232
- The playbook tests, app tests and asset related error will be fixed in the next mr
- The get_opt_params and min_number_log_statements will be fixed in the next mr

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright (c) Splunk Community, 2024
+   Copyright (c) 2016-2024 Splunk Inc.
```
Comment (alexa-phantom) [inline LICENSE:189]: I think we'd just make the copyrights 2024 and not 2016-2024

File: __init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# File: __init__.py
+#
+# Copyright (c) 2017-2024 Splunk Inc.
```
Comment (alexa-phantom) [inline __init__.py:3]: Fix

File: talosintelligence.json [hunk 3]
```diff
@@ -9,11 +9,12 @@
...
     "product_version_regex": ".*",
-    "publisher": "Splunk Community",
-    "license": "Copyright (c) Splunk Community, 2024",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2017-2024 Splunk Inc.",
     "app_version": "1.0.1",
     "utctime_updated": "2024-06-21T18:40:03.685771Z",
-    "package_name": "phantom_talosintelligence",
+    "package_name": "phantom_talosintelligencev2",
+    "fips_compliant": false,
```
Comment (alexa-phantom) [inline talosintelligence.json:17]: It would be good if this could be fips compliant...

File: talosintelligence.json [hunk 4]
```diff
@@ -58,7 +65,7 @@
             "action": "ip reputation",
             "identifier": "ip_reputation",
             "description": "Queries IP info",
-            "verbose": "",
+            "verbose": "Provides informations on an IP addresses reputation, enabling you to take proper action against untrusted, and unwated resources.",
```
Comment (alexa-phantom) [inline talosintelligence.json:68]:
```suggestion
            "verbose": "Provides information on an IP address's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```

File: talosintelligence.json [hunk 5]
```diff
@@ -132,7 +139,7 @@
             "action": "domain reputation",
             "identifier": "domain_reputation",
             "description": "Queries domain info",
-            "verbose": "",
+            "verbose": "Provides informations on a domains reputation, enabling you to take proper action against untrusted, and unwated resources.",
```
Comment (alexa-phantom) [inline talosintelligence.json:142]:
```suggestion
            "verbose": "Provides information on a domain's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```

File: talosintelligence.json [hunk 6]
```diff
@@ -208,7 +215,7 @@
             "action": "url reputation",
             "identifier": "url_reputation",
             "description": "Queries URL info",
-            "verbose": "",
+            "verbose": "Provides informations on an urls reputation, enabling you to take proper action against untrusted, and unwated resources.",
```
Comment (alexa-phantom) [inline talosintelligence.json:218]:
```suggestion
            "verbose": "Provides information on an url's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```

**Review by alexa-phantom — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (tapishj-splunk) [reply LICENSE]: Changed all the copyright dates

**Review by sodle-splunk — APPROVED** [review]

File: talosintelligence.json [hunk 7]
```diff
@@ -58,7 +68,7 @@
             "action": "ip reputation",
             "identifier": "ip_reputation",
             "description": "Queries IP info",
```
Comment (alexa-phantom) [inline talosintelligence.json:70]: More of a grammatical thing - your other actions are like "Test connectivity", "Validate the asset...", so follow same with this and the description on line 144
```suggestion
            "description": "Query IP info",
```

File: talosintelligence.json [hunk 8]
```diff
@@ -208,7 +218,7 @@
             "action": "url reputation",
             "identifier": "url_reputation",
             "description": "Queries URL info",
```
Comment (alexa-phantom) [inline talosintelligence.json:220]:
```suggestion
            "description": "Query URL info",
```

File: talosintelligence.json [hunk 9]
```diff
@@ -208,7 +218,7 @@
...
-            "verbose": "",
+            "verbose": "Provides information on an url's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```
Comment (alexa-phantom) [inline talosintelligence.json:221]: Do for all the other `verbose` where it is Provides-->Provide
```suggestion
            "verbose": "Provide information on an url's reputation, enabling you to take proper action against untrusted, and unwanted resources.",
```

**Review by alexa-phantom — CHANGES_REQUESTED** [review]

**Review by alexa-phantom — APPROVED** [review]

**Pull request merged**